DVB flexcop-pci: sanitize driver name to avoid warning on load

DVB driver for Flexcop/Technisat receiver currently calls request_irq()
with a long driver name containing slashes. This triggers a warning
when kernel tries to create /proc/irq/[NN]/[driver name] directory
(attached to avoid mangling).

The patch in attachment changes the driver name used in
pci_request_regions() and request_irq() to a shorter version without
slashes.

Signed-off-by: Jindrich Makovicka <makovick@gmail.com>

--- a/drivers/media/dvb/b2c2/flexcop-pci.c
+++ b/drivers/media/dvb/b2c2/flexcop-pci.c
@@ -38,7 +38,7 @@ MODULE_PARM_DESC(debug,
 	DEBSTATUS);
 
 #define DRIVER_VERSION "0.1"
-#define DRIVER_NAME "Technisat/B2C2 FlexCop II/IIb/III Digital TV PCI Driver"
+#define DRIVER_NAME "b2c2_flexcop_pci"
 #define DRIVER_AUTHOR "Patrick Boettcher <patrick.boettcher@desy.de>"
 
 struct flexcop_pci {
@@ -448,5 +448,5 @@ module_init(flexcop_pci_module_init);
 module_exit(flexcop_pci_module_exit);
 
 MODULE_AUTHOR(DRIVER_AUTHOR);
-MODULE_DESCRIPTION(DRIVER_NAME);
+MODULE_DESCRIPTION("Technisat/B2C2 FlexCop II/IIb/III Digital TV PCI Driver");
 MODULE_LICENSE("GPL");
